src/WebJobs.Extensions.IoTHub/Config/IoTCloudToDeviceExtension.cs

```diff
@@ -9,11 +9,13 @@ namespace Microsoft.Azure.WebJobs.Extensions.IoTHub.Config
 {
     public class IoTCloudToDeviceExtension : IExtensionConfigProvider
     {
-        private static string connectionString;
-        private static ServiceClient serviceClient;
+        private Dictionary<string, ServiceClient> _clients;
```

Comment:
>string [](start = 27, length = 6)

Can you include a comment for this field about what the key is?  'string' is ambiguous.